### PR TITLE
this is my favourite place

### PR DESCRIPTION
**Pull request by waiz93**

### CODE REVIEW

**Comment by mirasqn** [comment]
💯

File: _pins/waiz93.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+githubHandle: waiz93
+latitude: 52.489439
+longtitude:-1.890755
```
Comment (Champ0505) [inline _pins/waiz93.json]: no t on longitude!!!

**Comment by hollenberry** [comment]
Hey @waiz93 ! @Champ0505 is correct, and also you'll need to add some dashes to your code.

If you need any help, please check out the following video: https://vimeo.com/201011288, or reach out again and let us know if you have questions!

**Comment by crichID** [comment]
@waiz93 I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/6689/commits/7c09060d7bcb93649bff4e1992c57c1c19f07b19. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by May 26th, I will go ahead and do it just to keep the repository clean!

**Pull request merged**